frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -232,12 +246,16 @@ function Application({ getWidget }: DashboardProps): JSX.Element {
 								onClickHandler={(ChartEvent, activeElements, chart, data): void => {
 									onClickHandler(ChartEvent, activeElements, chart, data, 'Error');
 								}}
-								widget={getWidget([
-									{
-										query: `max(sum(rate(signoz_calls_total{service_name="${servicename}", operation=~\`${operationsRegex}\`, status_code="STATUS_CODE_ERROR"${resourceAttributePromQLQuery}}[5m]) OR rate(signoz_calls_total{service_name="${servicename}", operation=~\`${operationsRegex}\`, http_status_code=~"5.."${resourceAttributePromQLQuery}}[5m]))*100/sum(rate(signoz_calls_total{service_name="${servicename}", operation=~\`${operationsRegex}\`${resourceAttributePromQLQuery}}[5m]))) < 1000 OR vector(0)`,
-										legend: 'Error Percentage',
-									},
-								])}
+								widget={getWidgetQueryBuilder({
```

Comment:
same here also

Reply:
@palashgdev getWidgetQueryBuilder is again calling another function called errorPercentage. Did you mean we should memoize that function?

Reply:
it will be automatically asking to wrap in memo or callback